pkg/controller/pravegacluster/pravegacluster_controller.go

```diff
@@ -299,3 +310,39 @@ func (r *ReconcilePravegaCluster) syncControllerSize(p *pravegav1alpha1.PravegaC
 	}
 	return nil
 }
+
+func (r *ReconcilePravegaCluster) syncStatefulSetPvc(sts *appsv1.StatefulSet) error {
+	selector, err := metav1.LabelSelectorAsSelector(&metav1.LabelSelector{
+		MatchLabels: sts.Spec.Template.Labels,
+	})
+	if err != nil {
+		return err
```

Comment:
To be consistent, let's add an error message like here:

https://github.com/pravega/pravega-operator/pull/107/files#diff-d4ac6e435ae9f4544a5bc08fcf19c3ecR278
https://github.com/pravega/pravega-operator/pull/107/files#diff-d4ac6e435ae9f4544a5bc08fcf19c3ecR290

Reply:
Right I will modify that right now. Thanks